### PR TITLE
Adds offline UX guide

### PR DESCRIPTION
**Pull request by mustafa-x**
Added new article. Awaiting review.

### CODE REVIEW

**Comment by mustafa-x** [comment]
@petele Article is ready for your review :)

File: src/content/en/fundamentals/design-and-ui/offline-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: A guide to designing UX for slow networks and offline.
+
+{# wf_updated_on: 2016-11-10 #}
+{# wf_published_on: 2016-11-10 #}
+
+# From lie-fi to wi-fi: A guide to designing UX for slow networks and offline {: .page-title }
```
Comment (petele) [inline src/content/en/fundamentals/design-and-ui/offline-guide/index.md]: Title Should be Capital Case.
Subheadings should be sentence case

File: src/content/en/fundamentals/design-and-ui/offline-guide/index.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+{% include "web/_shared/contributors/mustafa.html" %}
+
+This article will take a look at multiple design considerations needed to create a great experience for users who have either a slow network connection or are completely offline.
```
Comment (petele) [inline src/content/en/fundamentals/design-and-ui/offline-guide/index.md]: Please break lines at 80 chars

File: src/content/en/fundamentals/design-and-ui/offline-guide/index.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+
+<figure class="attempt-left">
+  <img src="images/emojoy-toast-message.png" alt="Emojoy, the emoji messaging app informing the user when a change in state occurs.">
+  <figcaption>Clearly inform the user when a change in state occurs as soon as possible. </figcaption>
+</figure>
+<figure class="attempt-right">
+  <img src="images/io-toast-message.png" alt="The I/O 2016 app informing the user when a change in state occurs.">
+  <figcaption>The Google I/O app used a material design "toast" to let the user know when app was offline.</figcaption>
+</figure>
+
```
Comment (petele) [inline src/content/en/fundamentals/design-and-ui/offline-guide/index.md]: you probable want a `<div class="clearfix"></div>` after this

**Comment by mustafa-x** [comment]
Changes added :)

File: src/content/en/fundamentals/design-and-ui/offline-guide/index.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+description: A guide to designing UX for slow networks and offline.
+
+{# wf_updated_on: 2016-11-10 #}
+{# wf_published_on: 2016-11-10 #}
+
+# From Lie-fi To WI-FI: A Guide To Designing UX For Slow Networks and Offline {: .page-title }
+
+{% include "web/_shared/contributors/mustafa.html" %}
+
+This article will take a look at multiple design considerations needed to create a great experience on both slow networks and offline.
```
Comment (petele) [inline src/content/en/fundamentals/design-and-ui/offline-guide/index.md]: Please break lines at 80 chars

File: src/content/en/fundamentals/design-and-ui/offline-guide/index.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+* Think how you design for the success, failure and instability of a network connection
+
+* Data may be expensive, so be considerate to the user.
+
+* For most users globally,  the tech environment is almost exclusively mobile.
+
+* Low end devices are commonplace, with limited storage, memory and processing power and, small displays and lower touchscreen quality. Make sure performance is a part of your design process. 
+
+* Allow users to browser your application when they are offline.
```
Comment (jpmedley) [inline src/content/en/fundamentals/design-and-ui/offline-guide/index.md]: What do you mean by 'browser your application'?

**Comment by mustafa-x** [comment]
Sorry I meant browse, not browser :)

**Comment by petele** [comment]
@mustafa-x - please break lines at 80 characters.

**Comment by mustafa-x** [comment]
Hello mate

Is there a tool that can help you automate / count this? I've never written to a 80 char limit before :)

**Comment by petele** [comment]
Most text editors will put a ruler on the page at a configurable spot. 

In Sublime:
1. Place caret in the paragraph you want to break
2. Under the `Edit` menu, choose `Wrap` and then choose wrap at 80 or something below.

I'm sure other editors do it too, just not sure exactly how. You can also do it manually, figure out what about 80 characters looks like, and break. I'm not super strict about 80, but it's gotta be close.

**Pull request merged**